pb/clientmessage.proto

```diff
@@ -34,6 +40,7 @@ message OutboundMessage {
   uint32 nonce = 5;
   bytes block_hash = 6;
   repeated bytes signatures = 7;
+  repeated bytes payloads = 8;
```

Comment:
What is the difference between OutboundMessage.Payload and OutboundMessage.Payloads?

Reply:
Payloads is meant to replace Payload, similar to dests vs dest